core/src/main/java/org/apache/struts2/components/UIBean.java

```diff
@@ -796,28 +792,31 @@ public void evaluateParams() {
 
 
         // see if the value was specified as a parameter already
+        final String NAME_VALUE = "nameValue";
```

Comment:
Shouldn't this be class level constant with protected scope?

Reply:
I'm not sure, because only this method needs it - Again it's recommended by SonarLint to not repeat "nameValue" more than three times but only this method needs this constant!